### ISSUE TITLE
DatePicker: test often fails on travis

### ISSUE THREAD

**Issue by ilhan007** [SAP/ui5-webcomponents#5203]
### **Bug Description**
The "Keyboard navigation works when there are disabled dates in the calendar grid" test case is failing very often on the central Travis build

### **Expected Behavior**
The test should be stable.

### **Log Output / Stack Trace / Screenshots**
```
1) Date Picker Tests Keyboard navigation works when there are disabled dates in the calendar grid
[1] [chrome 101.0.4951.64 linux #0-18] Successfully navigated: expected false to be truthy
```
Related to: https://github.com/SAP/ui5-webcomponents/pull/5204

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-b please take a look of the test case as it fails often and blocks PRs from merging.
For the time being it will be commented. Make sure the test case is isolated properly - that may help.
